v4l2/v4l2_wrapper.cpp

```diff
@@ -206,3 +207,66 @@ int32_t YamiV4L2_SetDrmFd(int32_t fd, int drm_fd)
      return ret;
 }
 #endif
+
+int32_t YamiV4L2_SetParameter(int32_t fd, const char* key, const char* value)
+{
+    int ret = -1;
+
+    if (!key || !value) {
+        ERROR("invalue parameter\n");
+        return -1;
+    }
+
+    V4l2CodecPtr v4l2Codec = _findCodecFromFd(fd);
+    ASSERT(v4l2Codec);
+
+    if (!strcmp(key, "frame-memory-type")) {
```

Comment:
sorry, i'd better use english.
each mode has its value for now:
1. raw-pointer is similar to shm. if client dump video frames, it can used the data pointer to write output file. no additional copy.
2. raw-copy is similar to usr_ptr (copy indeed in our case), yami copy video frame data to user specific memory. then it's simple to manage the output buffer life cycle. not necessary for renderDone()
3. drm-name, it is friendly to other drm based modules, like mesa/opencl
4. dma_buf, it is friendly to camera, camera module has no knowledge on drm
5. surface-id, legacy support for the mode used by omxil, though i hate it. it brings vaapi complexity to omxil leve, is better be replaced by raw-pointer/row-copy